scaleway-python/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pulumi==3.38.0
+brlabs_scaleway==1.3.0
```

Comment:
Package name is not correct: should be `lbrlabs-scaleway`

https://pypi.org/project/lbrlabs-scaleway/

Reply:
aaaaaah, sorry! Done!